docs/json_schemas/fiveg_amf_n2/v0/provider.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "ProviderSchema",
+  "description": "Provider schema for fiveg_amf_n2.",
```

Comment:
I'd rename it to fiveg_n2. The interface is between the AMF and the gNodeB. I don't think we need/want the AMF name into the interface name.